Sources/LoopAlgorithm/LoopQuantity.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  File.swift
+//  LoopAlgorithm
```

Comment:
[nit] should this be `File.swift`?

Reply:
I'll update this. This is a real annoyance for me with Xcode 16 is the new default new file behavior.